crates/interpreter/src/side_table.rs

```diff
@@ -65,15 +61,15 @@ impl<'m> Metadata<'m> {
     }
 
     pub fn branch_table(&self) -> &[BranchTableEntry] {
-        bytemuck::cast_slice(&self.0[5 ..])
+        bytemuck::cast_slice(&self.0[10 ..])
```

Comment:
We still have the alignment problem here. BranchTableEntry shouldn't use `u16`, it should be `[u8; 6]`. And then we can remove the dependency on bytemuck.